Style.tsx

```diff
@@ -0,0 +1,179 @@
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignSelf: 'center',
+  },
+  TwitterLogo: {
```

Comment:
Always use camelcase for naming styles in stylesheet. Same goes for the other names below.